.github/workflows/build.yml

```diff
@@ -72,6 +75,49 @@ jobs:
       linux_matrix: ${{toJson(matrix.linux_matrix)}}
       windows_matrix: ${{toJson(matrix.windows_matrix)}}
 
+  real_storage_seed_linux:
```

Comment:
Think we can come up with a better name. 

These are regressions tests against persistant storage. How about calling it:

`reg_tests_data_seed_linux`

Reply:
How about just:
persistent_storage_seed_linux

Reply:
Sure, think that's a good name.